### PR TITLE
 Add configuration for the default sidebar panel and open state

### LINKED ISSUE #2402: Enable configuration of default state of window side panel
Related to #2372.

Currently, the About/Metadata panel is the panel that is shown when a window side panel is initially opened. 

However, an implementor might instead want the Rights panel to be the default panel displayed, because they are especially concerned that users see the rights/attribution info for the object. 

Issues to resolve:
- It is possible that an object manifest not include any rights/attribution info, in which case we intend that the Rights panel not be represented in the sidebar nav. But if an implementor configures the Rights panel to be the default, what happens when a user opens an object with no rights/attribution info (and thus there is no Rights panel to display)?

- For the use case where the implementor wants to configure the Rights panel as the default because they are especially concerned that users see the rights/attribution info for the object, is it enough to make the Rights panel the default when the sidebar is explicitly opened by the user, or do we also need to enable the implementor to indicate that the sidebar for a newly-opened window should be **open** by default?

Windows currently open with the sidebar closed, so even if the Rights panel is the default, there is no guarantee that the user will see the rights/attribution info, because they have to deliberately choose to open the sidebar to see it.

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #2402

### CODE REVIEW

File: src/config/settings.js [hunk 1]
```diff
@@ -180,8 +180,10 @@ export default {
   window: {
     allowClose: true, // Configure if windows can be closed or not
     allowMaximize: true, // Configure if windows can be maximized or not
+    defaultSideBarPanel: 'info', // Configure which sidebar is selected by default
```
Comment (camillevilla) [inline src/config/settings.js]:
```suggestion
    defaultSideBarPanel: 'info', // Configure which sidebar is selected by default. Options: info, rights, canvas, annotations
```

It would be nice to list the available panel names here. Another idea would be linking to [CompanionWindowFactory](https://github.com/ProjectMirador/mirador/blob/287e2ce1938cda8430a4b887c191066fbe5f50a1/src/components/CompanionWindowFactory.js#L17-L29)

Useful follow-up action would be renaming AttributionPanel.js so that it matches the `WindowSidebar{__}Panel` naming convention.

Comment (cbeer) [reply src/config/settings.js]: I'm trying to move us away from the `WindowSidebar{__}Panel` convention, it's way too wordy. I think we're now 1/2 and 1/2.
